de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1232,6 +1232,112 @@ image:chapterModeling/MaturityOverview.png[Maturity overview]
 
 Elements can be collapsed or expanded and the values can be exported to Excel. 
 
+
+==== State machine concept
+
+Virtual Satellite provides means to add state machines to your system but also of its single components. State machines describe the behavior of the system. It consists of a finite number of states and is therefore also called finite-state machine (FSM). Based on the current state and a given input the machine performs state transitions and produces outputs. 
+
+The basic building blocks of a state machine are:
+
+* `State:` 
+is a situation of a system depending on previous inputs and causes a reaction on following inputs. One state is marked as the initial state; this is where the execution of the machine starts.
+* `Transition:` defines for which input a state is changed from one to another. Depending on the state machine type, states and/or transitions produce outputs.
+* `AllowsConstraint:` should specify the states that are allowed to be activated while the parent state is active.
+* `ForbidsConstraint:` should specify the states that are forbidden while the parent state is active.
+
+In the following we will create a mass budget. Create a project - in the following called `MySatellite` - and activate the following concepts

```

Comment:
"we will create a mass budget" -> "we will create a state machine" ?

Reply:
changed